docs/compendium/substitute.md

```diff
@@ -27,13 +36,46 @@ input of request-body for substitution
 }
 ```
 
+**Example** request body with embedded file upload:
+
+```json
+{
+  "base": "G92NL",
+  "overlay": "9fCTR",
+  "substitutionFiles": [
+    {
+      "base": "climate-timeseries.csv",
+      "overlay": {
+        "filetype": "text/plain",
+        "filename": "substitution.txt",
```

Comment:
Shouldn't be the input file of same datatype of the base-file ?
In general there is not that difference between .csv and .txt, but in the substitution there it is right now important that the datatype is the same ( because this would cause the need of editing the main.Rmd where this file is gonna get read). Or am i wrong?

Reply:
Well, how do you know that the .txt file does not contain comma-seperated values?

I updated the example accordingly.